code/modules/flufftext/Hallucination.dm

```diff
@@ -269,51 +269,59 @@ Gunshots/explosions/opening doors/less rare audio (done)
 	px = -32
 
 /obj/effect/hallucination/oh_yeah
-	var/turf/simulated/wall/wall
-	var/obj/effect/hallucination/simple/bubblegum/bubblegum = null
+	var/obj/effect/hallucination/simple/bubblegum/bubblegum
```

Comment:
all these changes are entirely code improvements, no actual changes to behavior.